WordPress/Classes/ViewRelated/People/PersonViewController.swift

```diff
@@ -176,26 +176,38 @@ private extension PersonViewController {
 
         alert.addCancelActionWithTitle(cancelTitle)
 
-        alert.addDestructiveActionWithTitle(removeTitle) { action in
-            self.removePerson()
+        alert.addDestructiveActionWithTitle(removeTitle) { [weak self] action in
+            self?.deleteUser()
         }
 
         alert.presentFromRootViewController()
     }
 
-    func removePerson() {
+    func deleteUser() {
+        assert(user != nil)
```

Comment:
This is not really wrong at all, so feel free to ignore this comment.  It's very subjective.

I was thinking that maybe it'd make sense to just add `assertionFailure()` inside the guard below.  Up to you.